deps/download.sh

```diff
@@ -53,11 +53,12 @@ SQLITE_SOUNDEX
 # ========== START SCRIPT ========== #
 
 echo "setting up environment..."
-DEPS="$(CDPATH= cd -- "$(dirname -- "$0")" && pwd)"
+DEPS="$(node -e "console.log(require('path').resolve('.'))")"
```

Comment:
Can we change this to the more explicit `process.cwd()`?

Reply:
`cwd` doesn't resolve the working directory, so in general it's a better default, but you're going to be the one running this script the most...